src/nrniv/shape.cpp

```diff
@@ -1522,8 +1522,9 @@ void ShapeSection::set_range_variable(Symbol* sym) {
     bool any = false;
     if (nrn_exists(sym, section()->pnode[0])) {
         for (i = 0; i < n; ++i) {
-            pvar_[i] =
-                nrn_rangepointer(section(), sym, nrn_arc_position(section(), section()->pnode[i]));
+            assert(false);
```

Comment:
Another assert(false) cc: @nrnhines

Reply:
#2080 adds a test that covers this function.